chain/client/src/client_actor.rs

```diff
@@ -263,8 +263,7 @@ impl ClientActor {
     ) -> Res {
         let (_span, msg) = handler_debug_span!(target: "client", msg, msg_type);
         self.check_triggers(ctx);
-        let _d =
-            delay_detector::DelayDetector::new(|| format!("NetworkClientMessage {:?}", msg).into());
+        let _span_inner = tracing::debug_span!(target: "client", "NetworkClientMessage").entered();
```

Comment:
Hm, without the actual message printed, it becomes rather hard to figure out what is slow if this triggers.
Probably not a big problem. But if you have an idea how we can improve it, ideally without eagerly debug printing every time, it would be nice.
(If `#[track_caller]` worked with tracing spans and report the callers location instead of this one inside the `wrap`, I think that would be perfect. I don't know for sure but I think it doesn't work this way, however.)

Also, I think you can remove the `std::fmt::Debug` requirement on the function if you don't use debug formatting anymore.

Reply:
> what is slow if this triggers

That's when Grafana Tempo should be useful. It (should and will eventually) ingest the tracing information, and will let us search for long running "NetworkClientMessage" spans.